concent_api/verifier/tasks.py

```diff
@@ -47,6 +48,19 @@ def blender_verification_order(
     output_format: str,
     scene_file: str,  # pylint: disable=unused-argument
 ):
+    log_string_message(
+        logger,
+        f'Blender_verification_order_starts. SUBTASK_ID: {subtask_id} ',
+        f'Source_package_path {source_package_path} ',
```

Comment:
`log_string_message()` already joins the messages with a space in between. Adding a space here at the end will result in messages being separated with two spaces which will look weird.

Reply:
Right... done